.github/workflows/build-lint-test.yml

```diff
@@ -132,8 +132,22 @@ jobs:
           key: test-build-${{ runner.os }}-${{ matrix.node-version }}-${{ github.sha }}
       - run: yarn --immutable --immutable-cache
       - run: yarn workspace ${{ matrix.package-name }} run test:ci
-      - name: Upload coverage results
-        uses: codecov/codecov-action@d9f34f8cd5cb3b3eb79b3e4b5dae3a16df499a70
+      - name: Get coverage folder
+        id: get-coverage-folder
+        run: |
+          echo "stub" >> stub
```

Comment:
This is required because `actions/upload-artifact` takes the least common ancestor for determining where to upload the files. If we don't include a stub file at a different path outside of the coverage folder, it would use `coverage` as least common ancestor, and all `coverage-final.json` files would end up overwriting each other.